### DISCUSSION TITLE
After airflow upgrade " The site can´t be reached"

### DISCUSSION THREAD

**Discussion by aipatr** [apache/airflow#24697] [Q&A]
Dear all,
I upgrade from airflow 2.1 to 2.3.2 . The configuration is updated according to the new standard, server and scheduler are running on an Ubuntu machine with only Local workers but I get:
Some workers seem to have died and gunicorn did not restart them as expected

and the webserver is not reachable. How can I fix this? Thanks for your help

**Comment by aipatr** [comment]
Resolved, I needed to change the version of marshmallow. I believe that there should be a proper manual for upgrade between versions

**Reply by potiuk** [reply]
Which manualy did you use before @aipatr ? 

I am sure you are aware and checked before

*  the upgrade  manual here https://airflow.apache.org/docs/apache-airflow/stable/installation/upgrading.html  - 
and 
* the installation instructions  with https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html#installation-and-upgrade-scenarios which describes also the upgrade scenarios and explains that you MUST use constraints when you install/upgrade airflow.

And if you think that it could be descibed better - I heartiliy encourage you to update our docs to be clearer about it. Airflow is created by > 2100 contributors like you and contributing better docs is the best way to pay back for the software you use for free. Especially that - as a user who just struggled with it, you are the best to know what is missing and where you would look for the information. You are absolutely best candidate to improve the docs now.

This is super easy - eithe of these two pages has "Suggest a change on this page" button, that opens a PR where you can improve the docs using GitHub UI - no environment needed. It's as easy as writing that "there should be a proper manuai".

Looking forward to your contributions @aipatr !
